arrange_schedule.py

```diff
@@ -1049,7 +1049,128 @@ def check_event_exist_or_insert(event):
         with open(receive_msg["text_system_dir"],"w") as fp:
             print(json.dumps(text_file),file=fp)
 
-#crawler google drive image
+def find_drive_data_type():
+    db = mysql()
+    db.connect()
+    return_msg={}
+    return_msg['result'] = 'fail'
+    sql = "SELECT type_id FROM data_type WHERE type_name='google_drive_image'"
+    pure_result = db.query(sql)
+    try:
+        return_msg['data_type'] = int(pure_result[0][0])
+        return_msg['result'] = 'success'
+    except:
+        return_msg["error"] = "no google_drive_image data type"
+    db.close()
+    return return_msg
+
+def serach_google_drive_folder(service):
+    g_sql = "(name='1day' or name='3day' or name='7day' or name='14day')"
+    results = service.files().list(
+        q="mimeType='application/vnd.google-apps.folder' and trashed=false and " + g_sql, 
+        fields="nextPageToken, files(id, name)").execute()
+    items = results.get('files', [])
+    return items
+
+def serach_google_drive_file(service):
+    #set time
+    now_time = time.time()
+    start_time = time.strftime("%Y-%m-%dT%H:%M:%S+08:00", time.localtime(now_time-43200))
+    results = service.files().list(
+        q="modifiedTime > '" + start_time + "' and mimeType contains 'image/'", 
+        fields="nextPageToken, files(id, name, parents)").execute()
+    items = results.get('files', [])
+    return items
+    
+def merge_files_and_days(days_limit, drive_file):
+    for num1 in range(len(drive_file)):
+        drive_file[num1]['time'] = 86400*7
+        for item in days_limit:
+            if item['id'] in drive_file[num1]['parents']:
+                drive_file[num1]['time'] = 86400 * int(item['name'][:-3])
+                break
+    return drive_file
+
+def check_drive_img_exist(db, data_type, file_name):
+    sql = "SELECT COUNT(*) FROM image_data WHERE img_is_expire=0 and img_is_delete=0 "
+    sql = sql + "and type_id=" + str(data_type) + " and img_file_name='" + file_name + "'"
```

Comment:
Please modify the string usage to use `format` method.
Refer to [format example](https://docs.python.org/3.5/library/string.html#format-examples)